torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -1361,10 +1361,21 @@ Tensor binary_cross_entropy_target_backward(
   const Tensor& target,
   const c10::optional<Tensor>& weight,
   int64_t reduction) {
-  auto grad_target = (1. - self).log_().sub_(self.log()).mul_(grad);
+  auto grad_target = (1. - self).log_().sub_(self.log());
+  if (!areAnyTensorSubclassLike({grad})) {
+    // fast-path
```

Comment:
I am not sure such comment is useful.
If we want to explain why, we should have a note explaining when inplace are allowed and not right?

Reply:
Sure, I can remove the comments